{{cookiecutter.python_name}}/pyproject.toml

```diff
@@ -33,6 +33,7 @@ test = [{% if cookiecutter.test.lower().startswith('y') %}
     "pytest",
     "pytest-asyncio",
```

Comment:
You shouldn't need pytest-asyncio or pytest-tornasync if you're using pytest-jupyter.

Reply:
Actually, you'd need pytest-asyncio in order to write async fixtures.

Reply:
Thanks for the review @blink1073 

I removed `pytest-tornasync`